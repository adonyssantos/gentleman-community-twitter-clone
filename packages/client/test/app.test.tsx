import { create, ReactTestRenderer, ReactTestRendererJSON } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';
import { AppProvider } from '../src/providers/app.provider';

function toJson(component: ReactTestRenderer) {
  const result = component.toJSON();
  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);
  return result as ReactTestRendererJSON;
}

describe('@universal-labs/primitives', () => {
  it('Should render AppProvider', () => {
    const component = create(<AppProvider children={undefined} />);
    let tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
