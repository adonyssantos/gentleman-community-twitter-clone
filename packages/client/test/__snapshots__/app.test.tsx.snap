// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`@universal-labs/primitives > Should render AppProvider 1`] = `null`;
